Sources/PackageLoading/Manifest+parse.swift

```diff
@@ -40,18 +42,15 @@ extension Manifest {
         guard path.isFile else { throw PackageModel.Package.Error.noManifest(path) }
 
         // Load the manifest description.
-        if let toml = try parse(path: path, swiftc: swiftc, libdir: libdir) {
-            let toml = try TOMLItem.parse(toml)
-            let package = PackageDescription.Package.fromTOML(toml, baseURL: baseURL)
-            let products = PackageDescription.Product.fromTOML(toml)
-
-            self.init(path: path, package: package, products: products)
-        } else {
-            // As a special case, we accept an empty file as an unnamed package.
-            //
-            // FIXME: We should deprecate this, now that we have the `init` functionality.
-            self.init(path: path, package: PackageDescription.Package(), products: [])
+        guard let tomlString = try parse(path: path, swiftc: swiftc, libdir: libdir) else {
```

Comment:
👍 I appreciate the change to not reuse the name `toml` for both the string representation and the parsed form.